src/hotspot/share/classfile/classLoaderStats.hpp

```diff
@@ -112,7 +112,7 @@ class ClassLoaderStatsClosure : public CLDClosure {
   }
 
   typedef ResourceHashtable<oop, ClassLoaderStats,
-                            256, AnyObj::RESOURCE_AREA, mtInternal,
+                            256, AnyObj::C_HEAP, mtInternal,
```

Comment:
There's an mtStatistics NMT category, can you use that for this? and the instance below?

Reply:
Fixed.